pet-kata/src/test/java/org/eclipse/collections/petkata/Exercise4Test.java

```diff
@@ -13,60 +13,71 @@
 import java.util.AbstractMap;
 import java.util.Collections;
 import java.util.Comparator;
+import java.util.HashSet;
 import java.util.IntSummaryStatistics;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import org.eclipse.collections.api.list.MutableList;
 import org.eclipse.collections.impl.factory.Sets;
-import org.eclipse.collections.impl.list.mutable.FastList;
 import org.eclipse.collections.impl.test.Verify;
 import org.junit.Assert;
 import org.junit.Test;
 
 /**
  * In this set of tests, wherever you see .stream() replace it with an Eclipse Collections alternative.
+ * <p/>
+ * {@link org.eclipse.collections.api.list.primitive.MutableIntList}<br>
+ * {@link org.eclipse.collections.api.set.primitive.IntSet}<br>
+ * {@link org.eclipse.collections.impl.factory.primitive.IntSets}<br>
+ * {@link org.eclipse.collections.impl.block.factory.primitive.IntPredicates}<br>
+ * {@link org.eclipse.collections.api.bag.MutableBag}<br>
+ * {@link org.eclipse.collections.api.list.MutableList}<br>
+ *
+ * @see <a href="http://eclipse.github.io/eclipse-collections-kata/pet-kata/#/8">Exercise 4 Slides</a>
  */
 public class Exercise4Test extends PetDomainForKata
 {
     @Test
     public void getAgeStatisticsOfPets()
     {
+        Assert.fail("Refactor to Eclipse Collections");
```

Comment:
Should this comment be added here as is it still valid or useful ?  
// Don't forget to comment this out or delete it when you are done

Reply:
If you think it will not be obvious, sure.